support/dictionaries/dict-calcite-design-system.txt

```diff
@@ -43,6 +43,7 @@ Steffen
 signifiers
 stringifying
 uppercased
+milestoned
```

Comment:
Nitpick This could go under a `## process` or similar section too. I'll leave this up to you. ✨💪✨